echo-core/src/main/java/com/netflix/spinnaker/echo/DiscoveryActivator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.echo;
+
+import com.netflix.spinnaker.kork.discovery.InstanceStatus;
+import com.netflix.spinnaker.kork.discovery.RemoteStatusChangedEvent;
+import java.util.concurrent.atomic.AtomicBoolean;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
+public class DiscoveryActivator implements ApplicationListener<RemoteStatusChangedEvent> {
```

Comment:
you should be able to delete this and use the one from kork-core:

https://github.com/spinnaker/kork/blob/master/kork-core/src/main/java/com/netflix/spinnaker/kork/discovery/DiscoveryStatusListener.java#L24

Reply:
Nice, thank you! removing